src/app/base/hooks/analytics.ts

```diff
@@ -138,16 +138,19 @@ export const useGoogleAnalytics = (): boolean => {
       window.ga("set", "dimension1", version);
       window.ga("set", "dimension2", uuid);
 
-      sendPageview.current = () => {
-        const path = window.location.pathname + window.location.search;
-        if (path !== previousURL.current) {
-          window.ga("send", "pageview", path);
-          previousURL.current = path;
-        }
-      };
+      window.ga(
+        "send",
+        "pageview",
+        window.location.pathname + window.location.search
+      );
     }
   }, [allowGoogleAnalytics, authUser, uuid, version]);
 
+  useEffect(() => {
+    window.ga &&
+      window.ga("send", "pageview", location.pathname + location.search);
```

Comment:
While we need to use the React Router location to fire this useEffect we should send the window version: `window.location.pathname + window.location.search` otherwise the analytics will only get the subpath (e.g. "/machines").

Reply:
Sure, I'll update. But AFAIK this is what's currently sent to analytics anyway (domain is omitted).

Reply:
The difference is the React Router version omits the basename so you end up with "/machines" rather than "/MAAS/r/machines".